src/index.js

```diff
@@ -1,3 +1,62 @@
-function printToConsole() {
-  console.log("Hello!");
+const heroText = [
+  "measuring traffic density_",
```

Comment:
This has an implicit requirement that the text at a particular index here must match the image with the corresponding `hero-img-X` id. This isn't a big deal for a one-off, but usually in the long-term it'll be easier to maintain if we make that linkage explicit (it's pretty easy to imagine someone messing up the ordering if this is undergoing frequent changes).

Again, it's not necessary now (especially because it would be a fairly substantial refactor), but if doing this over again, I would probably structure this as an array of objects with text and img attributes:
```javascript
const heroContent = [
  {
    txt: "measuring traffic density_", 
    img: "images/hero-trafficdensity.jpg",
    alt: "Traffic on a highway interchange"
  }
...
]
```

And then inject that content into the DOM rather than cycling the visibility for a series of images.